log/log.go

```diff
@@ -0,0 +1,220 @@
+package log
+
+import (
+	"github.com/getsentry/raven-go"
+	"go.uber.org/zap"
+	"go.uber.org/zap/zapcore"
+)
+
+var (
+	logger = zap.NewNop().Sugar()
+)
+
+// Level is the verbose representation of log level
+type Level string
+
+// Enums for Level
+const (
+	NopLevel   Level = "nop"
+	DebugLevel Level = "debug"
+	InfoLevel  Level = "info"
+	WarnLevel  Level = "warn"
+	ErrorLevel Level = "error"
+	PanicLevel Level = "panic"
+	FatalLevel Level = "fatal"
+
+	// This will have the same effect as nop but slower
+	ZapNopLevel zapcore.Level = zapcore.FatalLevel + 1
+)
+
+// InitLogger provides a quick way to start or replace a logger
+// Pass in debug as log level enables development mode (which makes DPanicLevel logs panic).
+// Optionally pass in a cfg will provide a logger with custom setting
+func InitLogger(logLevel Level, cfg ...zap.Config) {
+	if logLevel == NopLevel {
+		logger = zap.NewNop().Sugar()
+		return
+	}
+
+	var config zap.Config
+
+	if len(cfg) > 0 {
+		config = cfg[0]
```

Comment:
It's kind of weird that we accept vararg but drop everything except the first one. Maybe we should have a `InitLoggerWithConfig(logLevel Lever, cfg zap.Config)` instead?

cc @andizzle

Reply:
Be more explicit in a lib is a good idea. 👍